app-modules/knowledge-base/src/Filament/Resources/KnowledgeBaseItemResource.php

```diff
@@ -55,7 +55,7 @@ class KnowledgeBaseItemResource extends Resource
 
     protected static ?string $breadcrumb = 'Knowledge Management';
 
-    protected static ?string $modelLabel = 'knowledge management';
+    protected static ?string $modelLabel = 'Knowledge Base Article';
```

Comment:
nit but this should be lower case if you want to follow the docs 100%

```suggestion
    protected static ?string $modelLabel = 'knowledge base article';
```